src/main/java/com/softserveinc/dokazovi/controller/EndPoints.java

```diff
@@ -12,6 +12,7 @@
 	public static final String POST_LATEST = "/latest";
 	public static final String POST_IMPORTANT = "/important";
 	public static final String POST_GET_POST_BY_ID = "/{postId}";
+	public static final String POST_DELETE_BY_ID = "/{postId}/delete";
```

Comment:
Sorry. Looks redundant. Please use the `DELETE` request with `/{postId}`.
Finally will be: `DELETE` request `localhost:8080/api/post/{postId}`